src/storage.js

```diff
@@ -0,0 +1,21 @@
+export const storageFactory = {
+  getItem(key) {
+    return JSON.parse(localStorage.getItem(key));
```

Comment:
What if the item does not exist?

Reply:
`localStorage.getItem` returns `null` and `JSON.parse(null) = null`